### PR TITLE
🚮 Assist.js: Remove assistjs extension and all relevant libraries.

### PR DESCRIPTION
**Pull request by ychsieh**
Assist.js AMP integration I2I issue: #30631(closing)

@samouri
/cc @ianba

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
Is there an I2R this should reference? Has any analysis been made on how many documents this may affect?

**Comment by ychsieh** [comment]
There shouldn't be analysis needed since this extension is not used anywhere.

**Comment by honeybadgerdontcare** [comment]
> There shouldn't be analysis needed since this extension is not used anywhere.

How have you come to that conclusion without any analysis?

What is the [I2R](https://github.com/ampproject/amphtml/blob/420bc3987f69f6d9cd36e31c013fc9eea4f1b245/docs/spec/amp-versioning-policy.md#deprecations) for this?

**Comment by ychsieh** [comment]
Just created: #35024. Thanks for the reminder.

**Comment by samouri** [comment]
@honeybadgerdontcare: this was a WIP extension that was completed. I don't believe it ever functioned either.

**Comment by honeybadgerdontcare** [comment]
> @honeybadgerdontcare: this was a WIP extension that was completed. It was never functional. At some point during implementation the team decided to end the effort, so this is just a cleanup.

Thank you for clarifying. Regardless this was something allowed and is now being disallowed. It should be verified that documents aren't using it even if it wasn't encouraged. I haven't seen anything stating this is not used in the open web.

**Comment by ychsieh** [comment]
Thanks for the kind scrutiny. This extension has always been under development and never entered into a usable stage, meaning all developers that tried to use it would either crashed their pages or found it non-responding. As a result, I don't think anyone is using it for serious purpose. Does that sound reasonable?

**Comment by honeybadgerdontcare** [comment]
@samouri for review

**Review by honeybadgerdontcare — APPROVED** [review]
approved for validator files

**Review by jridgewell — APPROVED** [review]

File: build-system/compile/bundles.config.extensions.json [hunk 1]
```diff
@@ -1233,4 +1225,4 @@
       "wrapper": "bento"
     }
   }
-]
+]
```
Comment (rsimha) [inline build-system/compile/bundles.config.extensions.json:1228]: nit: Restore trailing whitespace.

**Pull request merged**